### PR TITLE
Update event-bus image

### PR DESCRIPTION
**Pull request by montaro**
See also #3491

### CODE REVIEW

File: resources/event-bus/values.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ global:
     path: eu.gcr.io/kyma-project
   event_bus:
     dir: develop/
-    version: c157195c
+    version: a4acd811
```
Comment (marcobebway) [inline resources/event-bus/values.yaml]: This version does not include the changes done for https://github.com/kyma-project/kyma/issues/3491.

**Comment by montaro** [comment]
Updated to the latest version after merging https://github.com/kyma-project/kyma/pull/3592

**Pull request merged**